### PR TITLE
667 - separating QT code

### PR DESCRIPTION
**Pull request by HalcyonJAC**
**Author checklist**

- [x] Include primary ticket number in title
- [x] Fill in the details below and delete as appropriate
- [x] Be proactive in getting your work approved 💪

---

## What's included?
- Links to the Online Tests page should redirect to the new Online Tests website
- The existing Online tests page (on the Apply site) should still be accessible via its URL for now.
- The Online Tests page (and related components) have been deleted from the Apply site as part of this piece of development.

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
- Confirm that the Online Tests menu item redirects to the new website, instead of a local webpage
- Check that the /online-tests page can still be accessed on the apply site directly if the URL is entered manually into the browser's address bar

When testing, developers should checkout the qt-separation branch on the apply repo at the same time as the setup-beta-site branch on online-tests repo and run both sites at the same time. They should find that their browser switches between the two sites seamlessly:
- Apply = http://localhost:8081
- Online Tests = http://localhost:8083

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

---
PREVIEW:DEVELOP

### CODE REVIEW

File: src/views/Applications.vue [hunk 1]
```diff
@@ -11,6 +11,7 @@
               <RouterLink
                 class="govuk-link info-link--nav-applications--vacancies"
                 :to="{ name: 'vacancies' }"
+                data-cy="vacancies-link"
```
Comment (HalcyonJAC) [inline src/views/Applications.vue:14]: Note: These `data-cy` attributes are to assist with Cypress testing

**Review by warrensearle — CHANGES_REQUESTED** [review]
@HalcyonJAC please fix broken tests

**Review by warrensearle — APPROVED** [review]

File: vue.config.js [hunk 2]
```diff
@@ -14,7 +14,7 @@ module.exports = {
     ],
   },
   devServer: {
-    port: 8181,
+    port: 8081,
```
Comment (tomlovesgithub) [inline vue.config.js:17]: Can this stay 8181?

**Review by tomlovesgithub — APPROVED** [review]

**Comment by HalcyonJAC** [comment]
@warrensearle It seems that this file **src/views/QualifyingTests/QualifyingTest/Information.vue** which causes merge conflict will move to QT website after we separate QT code.

File: vue.config.js (hunk 2 above)
Comment (HalcyonJAC) [reply vue.config.js:17]: I changed it just for consistency. With the this change port numbers allocated across each of the front-end platforms would be allocated as follows:

admin - 8080
apply - 8081
assessments - 8082
qt - 8083

**Pull request merged**